types/rsmq-promise-native/rsmq-promise-native-tests.ts

```diff
@@ -0,0 +1,29 @@
+import RedisSMQPromise from "rsmq-promise-native";
+
+const rsmq = new RedisSMQPromise({host: "127.0.0.1", port: 7001, ns: "rsmq"});
+
+rsmq.createQueue({qname: 'myqueue'})
+    .then(done => {
+    	console.log('Queue created!');
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/rsmq-promise-native/rsmq-promise-native-tests.ts;
  Line: 7;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.